src/test/CMakeLists.txt

```diff
@@ -125,6 +125,7 @@ set(
     statistics/chunk_statistics/histograms/equal_distinct_count_histogram_test.cpp
     statistics/chunk_statistics/histograms/equal_height_histogram_test.cpp
     statistics/chunk_statistics/histograms/equal_width_histogram_test.cpp
+    statistics/chunk_statistics/histograms/histogram_utils_extended_test.cpp
```

Comment:
Move the tests into `histogram_utils_test.cpp`?

Reply:
Done.